### ISSUE TITLE
Shop search gets blocked by Privacy Badger

### ISSUE THREAD

**Issue by clem3ns** [EFForg/privacybadger#2109]
Our search (by Epoq @ www.asmc.de ) gets blocked by Privacy Badger.
Would be nice if Privacy Badger wouldn't block Epoq so our customers
can use the search in our onlineshop.

**Comment by ghostwords** [comment]
Is this a duplicate of #2108?

**Comment by ghostwords** [comment]
Could you provide the debugging information requested in https://github.com/EFForg/privacybadger/blob/1cac5ba752517802d0596645c464ffd55a60c9d0/.github/ISSUE_TEMPLATE/broken-site-report.md?

**Comment by clem3ns** [comment]
Yeah duplicated. Will close it
